pkg/cli/login/loginoptions.go

```diff
@@ -297,6 +333,46 @@ func (o *LoginOptions) gatherAuthInfo() error {
 	return nil
 }
 
+// prepareBuiltinExecPlugin sets up the ExecConfig correctly
+// with the given values
+func (o *LoginOptions) prepareBuiltinExecPlugin() (*kclientcmdapi.ExecConfig, error) {
+	issuerUrl, err := o.extractIssuerURLForOIDC()
+	if err != nil {
+		return nil, err
+	}
+
+	execProvider := &kclientcmdapi.ExecConfig{
+		APIVersion: clientauthentication.GroupName + "/v1",
+		Command:    "oc",
```

Comment:
wouldn't it make sense to use https://pkg.go.dev/os/exec#LookPath to get `oc` path?

Reply:
client-go executes plugins in here https://github.com/kubernetes/kubernetes/blob/f638e2849fbddeb201b8290586a4747af99086bf/staging/src/k8s.io/client-go/plugin/pkg/client/auth/exec/exec.go#L436 with `exec.Command`.

`exec.Command` already uses `LookPath` to resolve the path of plugin https://cs.opensource.google/go/go/+/master:src/os/exec/exec.go;l=359-361